### PR TITLE
feat: Hide code

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**
closes : https://github.com/zakodium/react-plot-docs/issues/1

### CODE REVIEW

**Comment by stropitek** [comment]
Can you push a dummy commit so that we get the netlify preview?

**Comment by wadjih-bencheikh18** [comment]
i didn't understand what is dummy commit

**Comment by stropitek** [comment]
> i didn't understand what is dummy commit

It's a commit that does nothing. I asked because it but should trigger netlify so that it builds the preview

```
git commit --allow-empty -m "dummy commit"
git push
```

**Comment by stropitek** [comment]
It's not very obvious that it's possible to click the header to show / hide the code. Could you make it a button instead?

**Comment by targos** [comment]
Maybe also add a chevron icon like in the menu?
image

File: src/theme/Playground/styles.module.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+.playgroundHeader {
+  letter-spacing: 0.08rem;
+  padding: 0.75rem;
+  text-transform: uppercase;
+  font-weight: bold;
+  background: var(--ifm-color-emphasis-600);
+  color: var(--ifm-color-content-inverse);
+  font-size: var(--ifm-code-font-size);
+  transition: max-height 1s ease-in-out;
```
Comment (stropitek) [inline src/theme/Playground/styles.module.css:24]: I think it'll be nicer if the animation does not take so long. Between 150ms and 300ms will make the UI feel snappier

**Comment by targos** [comment]
image

You don't need to create a dummy commit every time. It's just that the netlify service only triggers a build when something is pushed to the branch.

**Comment by lpatiny** [comment]
Not sure if this is the final proposal but I liked Michael proposal for

image

But also keeping the text 'Show live editor'

I would prefer the text is align left and not right

Also not sure why the code is now under the chart and not over the chart as before. It had the advantage that we were keeping the open / close bar over everything. I find it unusual to click on a button close to collapse something that is over the button.

**Comment by targos** [comment]
> Also not sure why the code is now under the chart and not over the chart as before. It had the advantage that we were keeping the open / close bar over everything. I find it unusual to click on a button close to collapse something that is over the button.

I agree. The button should not move when we click on it.

File: src/theme/Playground/styles.module.css (hunk 1 above)
Comment (wadjih-bencheikh18) [reply src/theme/Playground/styles.module.css]: fixed

**Comment by wadjih-bencheikh18** [comment]
chevron added

**Comment by targos** [comment]
I found two issues:
- If the live editor is collapsed, the changed code doesn't apply anymore
- there is a flickering effect when opening/collapsing

**Comment by wadjih-bencheikh18** [comment]
the problem in "@mdx-js/react" update 
many critical vulnerabilities apears when we use version  ^2.0.0 
with version 2.0.0-beta.15 of docusaurus

**Pull request merged**